gemfiles/railsmaster.gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-gem "rails", github: "rails/rails"
+gem "rails", "~> 7.0"
```

Comment:
This allows to fix [bugs](https://github.com/lioneldebauge/test-prof/actions/runs/7865021215/job/21457479005) related to a constant not being loaded in Rails. I fixed it by using only the last stable branch of version 7 but maybe there is a good reason why we don't do that already ?

Bugs was occuring both on my branch and on master

Reply:
Let's try adding `require "active_support/inflector"` somewhere here (before `factory_bot`): https://github.com/test-prof/test-prof/blob/02d8f355c158fb021e58ff1327d624a8299762b6/spec/support/ar_models.rb#L9

Reply:
@palkan done. CI is all green on my side 👍